src/structures/GuildAuditLogs.js

```diff
@@ -502,19 +502,21 @@ class GuildAuditLogsEntry {
           ),
         );
     } else if (targetType === Targets.INVITE) {
-      this.target = guild.members.fetch(guild.client.user.id).then(async me => {
-        if (me.permissions.has(Permissions.FLAGS.MANAGE_GUILD)) {
-          let change = this.changes.find(c => c.key === 'code');
-          change = change.new ?? change.old;
-          const invites = await guild.invites.fetch();
-          this.target = invites.find(i => i.code === change) ?? null;
-        } else {
-          this.target = this.changes.reduce((o, c) => {
-            o[c.key] = c.new ?? c.old;
-            return o;
-          }, {});
-        }
-      });
+      let change = this.changes.find(c => c.key === 'code');
+      change = change.new ?? change.old;
+
+      this.target =
+        guild.invites.cache.get(change) ??
```

Comment:
Is this fetch correct? type `AuditLogChange` doesn't seem to be the same as `Invite` which this cache stores.

Reply:
I noticed on create, delete & update, `data.target_id` is always `null`.

Here's an example output of fetching an entry of an invite that was created:

```
{
  id: 'id',
  user_id: 'user_id',
  target_id: null,
  action_type: 40,
  changes: [
    { key: 'code', new_value: 'code' },
    { key: 'channel_id', new_value: 'channel_id' },
    { key: 'inviter_id', new_value: 'inviter_id' },
    { key: 'uses', new_value: 0 },
    { key: 'max_uses', new_value: 0 },
    { key: 'max_age', new_value: 604800 },
    { key: 'temporary', new_value: false }
  ]
}
```

`code` there is the invite code. This seems to be the only way to grab the invite code, so:

```JavaScript
let change = this.changes.find(c => c.key === 'code');
change = change.new ?? change.old;
```
This should make `change` be the invite code. Then `<Guild>..invites.cache` is keyed by the invite code, so this should be fine. Unless I'm missing something here (which I hope not).